3rdparty/python/requirements.txt

```diff
@@ -2,6 +2,7 @@ ansicolors==1.1.8
 beautifulsoup4>=4.6.0,<4.7
 dataclasses==0.6
 fasteners==0.15.0
+libcst==0.3.12
```

Comment:
Nit, probably better to use `>=0.3.12,<0.4`. This will allow plugin authors to have slightly more flexibility and will allow us to get bug fixes whenever regenerating our constraints.txt.